docs/develop/java/testing-suite.mdx

```diff
@@ -311,6 +318,10 @@ Learn to skip time manually in the SDK of your choice.
 
 Learn to skip time in Activities in the SDK of your choice.
 
+#### Skip time in Nexus Operations {#skip-time-in-nexus-operations}
+
+Learn to skip time in Nexus Operations in the SDK of your choice.
```

Comment:
We don't support/can't support time skipping in nexus operations. I know this comment is what we say for activities, but it doesn't really make sense there either.